### ISSUE TITLE
7.0 UI/UX is convoluted and busy. It has no actions by default

### ISSUE THREAD

**Issue by umquat** [mozilla/multi-account-containers#1783]
- Multi-Account Containers Version: 7.0.0
- Operating System + Version: Linux Ubuntu 20.04
- Firefox Version: 78.0.1
- Other installed Add-ons + Version + Enabled/Disabled-Status:

### Actual behavior
To open a new tab in a chosen container I need to click on the  extension icon, then "Open new Tab in", then select the container.

### Expected behavior
Click on the extension icon, Click on a container name.

### Steps to reproduce
1.
2.
3.

### Notes

The old UI was fine. The default behavior of clicking on a container name is to open a new tab in that container. The new behavior is to ask me what to do in that container. 

I would suggest to define default behaviors and use another visual cue (ellipsis, etc) to overflow the other features.

In summary, please regard the primary goal of the extension to open a new tab in a container of my choosing and not manage all the possible options the extension has or will have. Managing options and customizing behavior is a transient, not constant activity while opening tabs is used many hundreds of times a day.

Thanks!

**Comment by FestTestQA** [comment]
me too

**Comment by eefi** [comment]
I contributed an idea in https://github.com/mozilla/multi-account-containers/issues/1784#issuecomment-657207449 for how to provide a simple "open a new tab in a container of my choosing" UI by keyboard shortcut while still keeping the complexity of actually managing containers available from the main extension UI.

Additionally, if you're used to using the mouse to open the Multi-Account Containers panel, then you can get the same workflow today by right-clicking on the new tab button (the plus at the right end of the tab strip). And if you enable the "Select a container for each new tab" option in about:preferences#containers , you can even get that menu to open up on a simple left-click.

**Comment by chriswalker01** [comment]
I agree with the previous comments. The new UI adds an unnecessary layer of complexity. Please allow us to set a default action (open new container) or make the new UI optional.

**Comment by gerryw1389** [comment]
Please bring this back. I use this extension daily for logging into different accounts and sold my IT department on this so one click container is essential. Yes we can use keyboard shortcuts but not everyone is comfortable with that.

**Comment by hackel** [comment]
Just to add another perspective, it always really annoyed me that the default action when clicking on a container was to open a new tab in that container, so the new UI changes are very welcome for me.  If I need to open a new tab in a new container, I'm either doing it with the keyboard, from the File -> New Container Tab menu, or link context menu, but this is extremely rare—usually only when I'm setting up container rules.

**Comment by aslmx** [comment]
Came here to "complain" about this as well ;)

my workflow was to click on the MutliAccountContainer Icon in the toolbar, scroll to the Container I'd liked to open and click on the "+" that appeared to open a container. 
Two Clicks, fine for me.
Now i need three clicks and need to scroll/move the mouse for the last click as well.

A default action or somehow fading in a "+" icon when you have the mouse positioned on a specific X position on a container row would be nice to have and would allow to have the other options appear with one click as well - as implemented now.

For the fun i'd like to mention this XKCD 
https://xkcd.com/1172/

**Comment by gerryw1389** [comment]
I agree with both of your responses. First, I think that right clicking a new tab icon and selecting a new container tab is a decent alternative to my old workflow. Secondly, I think that even if we stuck with the current UX design, @op still had the right idea of tucking these options behind an ellipsis or a fading '+' as @aslmx mentions.

**Comment by groovecoder** [comment]
This should be fixed in 7.0.1. We reverted the 1st click behavior to open a tab. The arrow/chevron goes to the sub-panel.
